package.json

```
@@ -11,7 +11,8 @@
   "license": "MIT",
   "private": false,
   "scripts": {
-    "build": "babel src --out-dir dist"
+    "build": "babel src --out-dir dist",
+    "release": "npm publish && git push origin head --tags"
```

One suggestion so all you need to do is run `yarn release` and not worry about whether you've done the build recently: 
```suggestion
    "release": "npm run build && npm publish && git push origin head --tags"
```